install_config/aggregate_logging.adoc

```diff
@@ -382,6 +383,25 @@ $ oc scale rc/logging-fluentd-1 --replicas=3
 
 You will need to scale Fluentd if nodes are added or subtracted.
 
+[NOTE]
+If the logging project was created using a node selector, then scaling Fluentd will fail. The scheduler will try to place all Fluentd replicas onto the same node. To fix this, edit the project:
```

Comment:
@adellape Actually, trying to have the two examples in the note box stuffs it up (too many ===='s). I just took out the [NOTE] bit. What's being said here doesn't exactly require it, and there was no way around it that I could find.